frontend/src/utilities/useWatchNotebooksForUsers.tsx

```diff
@@ -54,10 +54,13 @@ const useWatchNotebooksForUsers = (
             setLoadError(undefined);
           }
 
-          const newNotebooks = successes.reduce((notebookMap, { value }) => {
-            notebookMap[value.name] = value.data;
-            return notebookMap;
-          }, {} as UsernameMap<NotebookRunningState>);
+          const newNotebooks = successes.reduce(
+            (notebookMap, { value }) => ({
+              ...notebookMap,
+              [value.name]: value.data,
+            }),
+            {} as UsernameMap<NotebookRunningState>,
+          );
```

Comment:
Use variable name `acc`.
```suggestion
          const newNotebooks = successes.reduce((acc, { value }) => {
            acc[value.name] = value.data;
            return acc;
          }, {} as UsernameMap<NotebookRunningState>);
```